packages/kubernetes-1.25/kubelet-config

```diff
@@ -131,6 +131,7 @@ featureGates:
   CSIMigrationvSphere: true
 protectKernelDefaults: true
 serializeImagePulls: false
+seccompDefault: {{default false settings.kubernetes.seccomp-default}}
```

Comment:
I'm somewhat conflicted on this. On one hand, not storing anything for `seccomp-default` by default and letting it be set to false via the `default` helper is definitely simpler. But it's also nice to be able to query the Bottlerocket API to see whether `seccomp-default` is actually disabled or enabled or not. The API settings should probably be treated as declarative.

I'm leaning towards the `seccomp-default` be set to false by default explicitly and just render the value out as is here. Definitely would like more opinions on this.

Reply:
I would like to think that the Kubernetes default won't switch to _true_ in the middle of a release cycle. If for example they decide to make it _true_ for **1.30** we would already need to make a PR to make Bottlerocket's default _true_ as well. We could then set previous k8s variants explicitly to false in that same PR, but I could go either way if anyone has a strong feeling about doing it now.

Reply:
Very unlikely it would switch for a given release, more likely on a major.minor boundary.

This does mean we will need to watch out for that switching and update our default accordingly. It would be safer to not include the line unless it is explicitly set by the user. But I don't feel too strongly about it in this case.

Reply:
I went ahead and added a commit to explicitly set `settings.kubernetes.seccomp-default` to `false` in the datastore to make this more clear. Whenever we want to switch to `true` for the default, we can make a **kubernetes-seccomp-default-true.toml** and point newer variants to that instead.